site/app/controllers/UserProfileController.php

```diff
@@ -145,6 +145,29 @@ public function changePronouns() {
         }
     }
 
+    /**
+     * @Route("/user_profile/change_display_name_order", methods={"POST"})
+     * @return JsonResponse
+     */
+    public function changeDisplayNameOrder() {
+        $user = $this->core->getUser();
+        if (isset($_POST['display-name-order'])) {
+            $newPronouns = trim($_POST['display-name-order']);
+            //validPronouns() checks for valid option
```

Comment:
The comments here can also go unless they are essential for a valid display order, which should be validated here in the backend.